.rubocop.yml

```diff
@@ -62,9 +65,15 @@ Style/EachWithObject:
 Style/Encoding:
   Enabled: false
 
+Style/EmptyLinesAroundClassBody:
```

Comment:
When I disabled this and ran it, all it did was raise a single warning about some new line spacing. I think keeping it on will enforce some style consistency.